src/guiguts/file.py

```diff
@@ -508,6 +510,9 @@ def interpret_bin(self, bin_dict: BinDict) -> bool:
         if bookmarks := bin_dict.get(BINFILE_KEY_BOOKMARKS):
             for key, value in bookmarks.items():
                 self.set_bookmark_index(key, value)
+        image_rotations: Optional[dict[str, int]]
+        if image_rotations := bin_dict.get(BINFILE_KEY_IMAGEROTATE):
+            mainimage().rotation_details = image_rotations
```

Comment:
I think this needs something like the following to clear the previous file's rotations when the newly loaded dict doesn't have any image rotations:
```python
else:
    mainimage().rotation_details = {}
```

Reply:
Good catch, that's exactly it. New version pushed.